lib/reply.js

```diff
@@ -612,6 +613,7 @@ function setupResponseListeners (reply) {
   reply[kReplyStartTime] = now()
 
   const onResFinished = err => {
+    reply[kReplyEndTime] = now()
```

Comment:
set this only if `this[kReplyStartTime]` is set. Getting the time is relatively expensive.

Reply:
I don't understand how it could not be set. As it is set in the parent function.

Reply:
Ah sorry, I didn't see it!